gilbreth_support/urdf/break_beam/break_beam.gazebo.xacro

```diff
@@ -18,8 +18,8 @@
             <horizontal>
               <samples>50</samples>
               <resolution>1</resolution>
-              <min_angle>-${radians(20)}</min_angle>
-              <max_angle>${radians(20)}</max_angle>
+              <min_angle>0</min_angle>
+              <max_angle>0</max_angle>
```

Comment:
How can the min and max angles be the same?

Reply:
It seems unnatural, but in this situation, the break beam only has one beam. Otherwise, it has multiple beams.